client/resource_group/controller/config.go

```diff
@@ -89,9 +91,17 @@ func defaultRequestUnitConfig() *RequestUnitConfig {
 		WriteBaseCost:    defaultWriteBaseCost,
 		WriteCostPerByte: defaultWriteCostPerByte,
 		CPUMsCost:        defaultCPUMsCost,
+		IsServerless:     false,
 	}
 }
 
+// DefaultServerlessRUConfig returns the default configuration of Serverless.
+func DefaultServerlessRUConfig() *RequestUnitConfig {
```

Comment:
```suggestion
func SetSingleGroupByKeyspaceMode() *RequestUnitConfig {
```
It's better not to distinguish between cloud and op